dsc_lib/src/dscerror.rs

```diff
@@ -166,11 +167,11 @@ impl StreamMessage {
     pub fn print(&self, error_format:&StreamMessageType, warning_format:&StreamMessageType) -> Result<(), DscError>{
         if self.message_type == StreamMessageType::Error
         {
-            eprintln!("{:?} -> {} {}", error_format, self.resource_type_name, self.message);
+            error!("{:?} -> {} {}", error_format, self.resource_type_name, self.message);
         }
         if self.message_type == StreamMessageType::Warning
         {
-            eprintln!("{:?} -> {} {}", warning_format, self.resource_type_name, self.message);
+            error!("{:?} -> {} {}", warning_format, self.resource_type_name, self.message);
```

Comment:
should this be using `warning` instead of `error`?  For now, we can have subscriber write warnings to stderr.